.changeset/green-cheetahs-rush.md

```diff
@@ -0,0 +1,11 @@
+---
+"@itwin/itwinui-css": minor
+---
+
+`iui-anchor` can now be styled with status colors using a new `data-iui-status` attribute.
```

Comment:
To confirm, are we intending to use the status only when the anchor is used inside another component? Or are we intending it to be used on standalone anchors too?

If we don't intend this to be used on standalone anchors, maybe we should mention this in the changeset?

Also, in general, should we mention something regarding the intended use-cases of this to prevent overruse of anchors with statuses?

Reply:
The primary use case for this attribute is inside of the `Alert` and `Toast` components, but I don't think that means that developers should be cautioned from using this attribute with standalone anchors, if they feel like it. The attribute only changes the anchor's color, so I don't think overuse is much of a concern either. Mentioning the primary use case might make it more clear why we are adding this attribute, but I don't want to accidently make the changeset unclear as a result.

Reply:
This changeset is for the CSS package, which is really only used by us. We are not adding this as a prop in React, so there is no chance of misuse.

You can compare the download numbers of the recent versions:
- css: https://www.npmjs.com/package/@itwin/itwinui-react?activeTab=versions
- react: https://www.npmjs.com/package/@itwin/itwinui-react?activeTab=versions